pkg/controller/storagecluster/reconcile.go

```diff
@@ -25,6 +25,10 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/reconcile"
 )
 
+// ReconcileStratery is a string representing how we want to reconcile
+// (or not) a particular resource
+type ReconcileStratery string
```

Comment:
`s/ReconcileStatery/ReconcileStrategy/` everywhere?

Reply:
HAH! At least the typo was consistent ;D Fixed.

Reply:
yeah, it built...